deploy/kubernetes/rbac.yaml

```diff
@@ -24,6 +24,9 @@ rules:
   - apiGroups: [""]
```

Comment:
I think the events and nodes rules are not needed.   Also, can you change the name of the cluster role to something like "cluster-driver-registrar-role"